src/reset-password/controller/redirect.ts

```diff
@@ -9,6 +9,12 @@ import { resetPasswordForm } from '../formats/redirect';
 
 class ResetPasswordController extends Controller {
 
+  /**
+   * Currently, this GET request isn't being used since there is a browser issue where
+   * it gets redirected from the email link, the browser clears the cookie
+   *
```

Comment:
We *do* actually use this endpoint, when a user submits something and the new passwords didn't match.